hooks/pre_validations/12-check_capsule_tar.rb

```diff
@@ -4,12 +4,19 @@ def error(message)
   kafo.class.exit 101
 end
 
-certs_tar = param('foreman_proxy_certs', 'certs_tar') ||
-  param('foreman_proxy_content', 'certs_tar')
+ssl_directory = param('certs', 'ssl_build_dir')
+certs_tar = param('foreman_proxy_certs', 'certs_tar') || param('foreman_proxy_content', 'certs_tar')
 
 if certs_tar.value
   certs_tar.value = File.expand_path(certs_tar.value)
-  if param('certs', 'deploy').value && !File.file?(certs_tar.value)
+end
+
+# On existing installs, check to see if ssl_directory exists and if so skip the certs_tar value check
+if File.file?('./config/last_scenario.yaml')
```

Comment:
This will fail in production. You may be able to use `Dir.join(scenario[:installer_dir], 'config', 'last_scenario.yaml')` but I'm wondering if this is really a good check.